### PR TITLE
Day of dashboard

### PR DESCRIPTION
**Pull request by rosslh**
## Proposed Change

Create placeholder day of dashboard

### How did you do this?

Follow the mocks minus the sidebar

### Why are you choosing this approach?

Hackers need a landing page after they apply!

### Any open questions you want to ask code reviewers?

No

### List of changes:

  - Create Event component
  - Create Dashboard
  - Create DashboardMenu

## Pull Request Checklist:

  - [x] My submission passes all tests.
  - [x] I have lint my code locally prior to submission.
  - [ ] I have written new tests for my core changes (where applicable).
  - [x] I have referenced all useful information (issues, tasks, etc) in the pull request.

### CODE REVIEW

File: packages/client/components/Dashboard/Card/Card.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+
+class Class extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
```
Comment (RobertWSaunders) [inline packages/client/components/Dashboard/Card/Card.js]: No need for state here?

File: packages/client/components/Dashboard/Card/Card.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+
+class Class extends Component {
```
Comment (RobertWSaunders) [inline packages/client/components/Dashboard/Card/Card.js]: This should be `Card`!

File: packages/client/components/Dashboard/Card/Card.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            {this.props.title}
+          </h3>
+          <div>{this.props.children}</div>
+        </div>
+      </div>
+    );
+  }
+}
+
+export default Class;
```
Comment (RobertWSaunders) [inline packages/client/components/Dashboard/Card/Card.js]: This should be `Card`!

File: packages/client/components/Dashboard/DashboardMenu.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+              line-height: 44px;
+            `}
+          >
+            <ActionButton
+              style={`font-size: 16px;`}
+              type="rounded"
+              internal
+              link="/logout"
+            >
+              Log out
```
Comment (RobertWSaunders) [inline packages/client/components/Dashboard/DashboardMenu.js]: Lets make sure this uses the React Router `Link` when you make that change.

Comment (rosslh) [reply packages/client/components/Dashboard/DashboardMenu.js]: I've got this as a change in my other PR

Comment (RobertWSaunders) [reply packages/client/components/Dashboard/DashboardMenu.js]: Awesome!

**Pull request merged**